packages/client/src/runtime/RequestHandler.ts

```diff
@@ -149,7 +150,7 @@ export class RequestHandler {
             document,
             runInTransaction,
           },
-          (params) => this.dataloader.request(params),
+          (params) => this.dataloader.request({ ...params, tracingConfig: this.client._tracingConfig }),
```

Comment:
I think this area of the code is barely tested, and unused (see all the missing args). I think we should totally remove it at some point, it's not part of the public API.

Reply:
This is tested here: https://github.com/prisma/prisma/blob/main/packages/client/src/__tests__/integration/happy/middlewares/test.ts#L54